src/components/common/Input/Input/Input.jsx

```diff
@@ -123,17 +123,19 @@ class Input extends Component {
         <label htmlFor={id} className={`${styles.label} ${this.hasInput && styles.hasinput}`}>
           { label || placeholder }
         </label>
-        { children }
-        { clearable && (
-          <MaterialIcon
-            icon="cancel"
-            className={`${styles.clearbutton} ${this.hasInput && styles.hasinput}`}
-            onClick={this.clear}
-            tabIndex="0"
-            role="button"
-            title="Clear input field"
-          />
-        )}
+        <div className={styles.buttonsContainer}>
+          {children}
```

Comment:
add spacing around children, to match style in the rest of this file 

(a bit nitpicky, and I know I said the opposite in another place. But IMO it's most important that we stay consistent within a single file)